src/toDo.js

```diff
@@ -48,7 +60,8 @@ export default class ToDo {
   }
 
   removeItem() {
-    this.remove.addEventListener('click', () => {
+    this.remove.addEventListener('click', (e) => {
+      e.preventDefault();
       this.toDo = this.toDo.filter((item) => item.completed === false);
       localStorage.setItem('to-do', JSON.stringify(this.toDo));
       this.listHolder.innerHTML = '';
```

Comment:
- [ ] Please implement the remove function to remove a single particular item from the list, actually, the only working is `remveAllCompleted`, kindly take a look at [this](https://www.youtube.com/watch?v=AcUd-_Yjjqg)
  - the selected item should have a `line through` it showing that it is completed
  - you may be able to delete that particular item from the list.
  
<img width="632" alt="select" src="https://user-images.githubusercontent.com/79058364/159916314-a8172b9c-f209-4732-a3de-b4b595eb7aa2.png">